modules/update-mco-process.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="mco-update-process_{context}"]
+= Machine Config Operator node updates
+The Machine Config Operator (MCO) applies a new machine configuration to each control plane and compute node. The `.spec.maxUnavailable` configuration, which has a default value of one node, determines how many nodes can simultaneously undergo the update process at any given time.
+
+When the machine configuration update process begins, the MCO checks the amount of currently unavailable nodes in a pool. If there are fewer unavailable nodes than the value of `.spec.maxUnavailable`, the MCO will initiate the following sequence of actions on available nodes in the pool:
+
+. Cordon and drain the node
++
+[NOTE]
+====
+When a node is cordoned, workloads cannot be scheduled to it.
+====
+
+. Update the system configuration and operating system (OS) of the node
+
+. Reboot the node
+
+. Uncordon the node
+
+A node undergoing this process is unavailable until it is uncordoned and workloads can be scheduled to it again. The MCO begins updating nodes until the number of unavailable nodes is equal to the value of `.spec.maxUnavailable`.
+
+As a node completes its update and becomes available, the number of unavailable nodes is once again fewer than `.spec.maxUnavailable`. If there are remaining nodes that need to be updated, the MCO initiates the update process on a node until the `.spec.maxUnavailable` limit is once again reached. This process repeats until each control plane and compute node has been updated.
+
+The following example workflow describes how this process might occur in a 5 node pool, where `.spec.maxUnavailable` is 3 and all nodes are initially available:
```

Comment:
`node pool` is not accurate. it should be mcp(machine config pool). master or worker